src/Storages/ObjectStorage/StorageObjectStorage.cpp

```diff
@@ -489,10 +523,17 @@ void StorageObjectStorage::Configuration::initialize(
 
     if (configuration.format == "auto")
     {
-        configuration.format = FormatFactory::instance().tryGetFormatFromFileName(
-            configuration.isArchive()
-            ? configuration.getPathInArchive()
-            : configuration.getPath()).value_or("auto");
+        if (configuration.isDataLakeConfiguration())
+        {
+            configuration.format = "Parquet";
```

Comment:
I believe data lakes may have files in different formats (parquet, orc, avro). And it would be good to move it to the method of configuration `configuration.getDefaultFormatName()` that can even look at the metadata to check the actual format used in files